lib/capybara.rb

```diff
@@ -66,39 +66,145 @@ class << self
     #       config.app_host   = 'http://www.google.com'
     #     end
     #
-    # === Configurable options
-    #
-    # [app_host = String/nil]             The default host to use when giving a relative URL to visit, must be a valid URL e.g. http://www.example.com
```

Comment:
[question] A description about `app_host` default value is missing, is it no problem?

Reply:
It defaults to nil - so it's fine